features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.js

```diff
@@ -95,12 +95,14 @@ class Details extends React.Component {
         let existing_subscriptions = api.getSubscriptions(this.api_uuid, null);
         let promised_applications = api.getAllApplications();
 
-        Promise.all([ promised_api, existing_subscriptions, promised_applications] ).then(
+        //Promise.all([ promised_api, existing_subscriptions, promised_applications] ).then(
```

Comment:
Shall we remove the commented lines?

Reply:
The file has been updated with new logic and replaced with [index.jsx](https://github.com/wso2/carbon-apimgt/blame/6.5.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.jsx)